components/prism-core.js

```diff
@@ -527,11 +567,11 @@ if (!_self.document) {
 }
 
 //Get current script and highlight
-var script = document.currentScript || [].slice.call(document.getElementsByTagName('script')).pop();
```

Comment:
Is it worth using this `pop` fallback if we don't find the current script in IE11?

Reply:
The whole point of this PR is that this method doesn't work.

I think it's better to say that we couldn't find a `script` instead of returning an incorrect one.

Reply:
Ok, I agree with that.